bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -192,13 +216,21 @@
 		<label>Occupancy</label>
 		<description>Occupancy button pressed or released.</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Presence</tag>
+		</tags>
```

Comment:
Think these tags should be removed as it does not seem to fit. Weird channel anyway.